Packet++/src/TextBasedProtocol.cpp

```diff
@@ -108,6 +108,7 @@ void TextBasedProtocolMessage::parseFields()
 	// last field can be one of:
 	// a.) \r\n\r\n or \n\n marking the end of the header
 	// b.) the end of the packet
+	int iterations = 0;
```

Comment:
it'd be useful to add a comment which explains why we need to limit to 100 iterations

Reply:
I have added a comment and a reference to the Apache documentation of the LimitRequestFields Directive.